src/components/NavBar/utils.tsx

```diff
@@ -0,0 +1,71 @@
+import { chain, map, startCase, camelCase, mapKeys } from 'lodash';
+
+import appJSON from '../../config/prism.json';
+import layersJSON from '../../config/layers.json';
+import baseline from '../images/icon_basemap.png';
+import climate from '../images/icon_climate.png';
+import impact from '../images/icon_impact.png';
+
+const icons: { [key: string]: string } = {
+  baseline,
+  climate,
+  impact,
+};
+
+export interface ILayer {
```

Comment:
I suspect this is a type that is going to be used widely in the App (the data processing sections of the page will also need access to it). Let's not define it in a `NavBar` specific path.

Maybe `src/config/types.ts`?